zipline/finance/performance/tracker.py

```diff
@@ -77,7 +78,7 @@ class PerformanceTracker(object):
     """
     Tracks the performance of the algorithm.
     """
-    def __init__(self, sim_params, trading_calendar, env):
```

Comment:
I'm pretty sure the only thing we're using `env` for is to get the asset finder, which is also on the data portal.